cpp/arcticdb/version/local_versioned_engine.hpp

```diff
@@ -281,6 +281,8 @@ class LocalVersionedEngine : public VersionedEngine {
 
     timestamp get_update_time_internal(const StreamId &stream_id, const VersionQuery &version_query);
 
+    VersionedItem refresh_symbol_cache();
```

Comment:
Even internal to the C++ I don't think `symbol_cache` is a clear name. In fact even though it's not necessary, I'm inclined to think in this case that `symbol_stats_cache` should be the consistent name used throughout both the Python and C++.